src/types/models/index.ts

```diff
@@ -180,6 +180,16 @@ export interface SearchCallLead {
   duration: number
 }
 
+export interface SearchWhatsAppLead {
```

Comment:
the other whatsapp type/function is written as `Whatsapp` and not `WhatsApp` can we please be consistent and either adjust the old one or name the new one similar? Thanks!